data/newMigrations/016-genders.js

```diff
@@ -0,0 +1,10 @@
+exports.up = (knex) => {
+    return knex.schema.createTable('genders', function (tbl) {
+        tbl.increments('gender_id').primary();
```

Comment:
`.primary()` is not needed, when using `.increments()` knex sets it as primary behind the scenes